<html>
    <script>
        class CustomError extends Error
        {
            constructor(msg)
            {
                super(msg);
            }
        }

        try{
            throw new CustomError("Custom error")
        }

        catch(e)
        {
            console.log("E", e)
        }
    </script>
</html>